docs/cli_help.md

```diff
@@ -133,6 +133,23 @@
   * Applies to the most recent `--benchmark`, or all benchmarks if specified
     before any `--benchmark` arguments.
 
+* `--discard-on-throttle`
```

Comment:
I think we should just make this the default behavior. I don't think we need an opt-out -- if throttle checks are enabled, always discard throttled results.